src/Evolution/Executables/Burgers/EvolveBurgers.hpp

```diff
@@ -189,12 +188,37 @@ struct EvolutionMetavars {
 
   enum class Phase {
     Initialization,
+    LoadBalancing,
     RegisterWithObserver,
     InitializeTimeStepperHistory,
     Evolve,
     Exit
   };
 
+  static std::string phase_name(Phase phase) noexcept {
+    if (phase == Phase::LoadBalancing) {
+      return "LoadBalancing";
+    }
+    ERROR("Passed phase that should not be used in input file");
```

Comment:
[optional] maybe add `. Integer corresponding to phase is: " << static_cast<int>(phase)`